src/copy.jl

```diff
@@ -90,3 +90,222 @@ function defaultcopy!(dest::MOI.AbstractInstance, src::MOI.AbstractInstance)
 
     return MOI.CopyResult(MOI.CopySuccess, "", idxmap)
 end
+
```

Comment:
Please add a comment explaining the motivation for this interface.

Reply:
I have added it in 8b3db80